Emby.Server.Implementations/Net/UdpSocket.cs

```diff
@@ -236,35 +233,40 @@ public IAsyncResult BeginSendTo(byte[] buffer, int offset, int size, IpEndPointI
 
             var ipEndPoint = NetworkManager.ToIPEndPoint(endPoint);
 
-            return _Socket.BeginSendTo(buffer, offset, size, SocketFlags.None, ipEndPoint, callback, state);
+            return _socket.BeginSendTo(buffer, offset, size, SocketFlags.None, ipEndPoint, callback, state);
         }
 
         public int EndSendTo(IAsyncResult result)
         {
             ThrowIfDisposed();
 
-            return _Socket.EndSendTo(result);
+            return _socket.EndSendTo(result);
         }
 
-        protected override void Dispose(bool disposing)
+        private void ThrowIfDisposed()
```

Comment:
who's going to call this function?

Reply:
It's already being used by the internal functions.